core/shared/platform/common/posix/posix_time.c

```diff
@@ -15,3 +16,142 @@ os_time_get_boot_microsecond()
 
     return ((uint64)ts.tv_sec) * 1000 * 1000 + ((uint64)ts.tv_nsec) / 1000;
 }
+
+uint64
+os_clock_res_get(bh_clock_id_t clock_id, uint64 *resolution)
+{
+    #ifdef BH_PLATFORM_WINDOWS 
```

Comment:
I think we should only keep the posix implementation in this file; windows implementation should be in the `win_time.c`